packages/connect-thegraph-agreement/subgraph/src/Agreement.ts

```diff
@@ -0,0 +1,297 @@
+import { ERC20 } from '../generated/schema'
+import { ERC20 as ERC20Contract } from '../generated/templates/Agreement/ERC20'
+import { ethereum, BigInt, Address } from '@graphprotocol/graph-ts'
+import { Agreement, Action, Signature, Version, Disputable, Challenge, Dispute, Evidence, Signer, CollateralRequirement } from '../generated/schema'
```

Comment:
I like the initiative! The only thing that could be confusing is the `AgreementEntity`, especially since it is not the same as the entities exposed by the connector, these are subgraph entities

Reply:
Yes, that's a good point. In my opinion, having the distinction on the mappings clarify regardless. Maybe we can think on a better naming to the connectors entities?

Reply:
Yea I agree, what about `models` for the connector entities instead? It's just a synonym, but it will help to clarify

Reply:
Great, I like it. I can refactor to the new naming on a new PR.

Reply:
Just wanted to confirm that we are only referring to subgraph-related code here? I would avoid using suffixes for entities [at the library level](https://github.com/aragon/connect/tree/agreement_subgraph/packages/connect-agreement/src/entities) since these are exposed to users.

Reply:
Yup, my intention was to use synonyms for the subgraph entities, but also rename the `entities` dir in the connector to `models` to avoid confusing these